docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -78,7 +77,7 @@ The Telemetry Operator watches all LogPipeline resources and related Secrets. Wh
     An output is a data destination configured by a [Fluent Bit output](https://docs.fluentbit.io/manual/pipeline/outputs) of the relevant type. The LogPipeline supports the following output types:
 
     - **http**, which sends the data to the specified HTTP destination. The output is designed to integrate with a [Fluentd HTTP Input](https://docs.fluentd.org/input/http), which opens up a huge ecosystem of integration possibilities.
-    - **grafana-loki**, which sends the data to the Kyma-internal Loki instance. Note: This output is considered legacy and is only provided for backwards compatibility with the in-cluster Loki instance. It might not be compatible with latest Loki versions. For integration with Loki, use the `custom` output with name `loki` instead.
+    - **grafana-loki**, which sends the data to the Kyma-internal Loki instance. Note: This output is considered legacy and is only provided for backwards compatibility with the [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) in-cluster Loki instance. It might not be compatible with latest Loki versions. For integration with Loki, use the `custom` output with name `loki` instead, see also [this tutorial](https://github.com/kyma-project/examples/tree/main/loki).
```

Comment:
Not sure if we should provide a link to install Loki here. The sentence does not sound complete.
```suggestion
    - **grafana-loki**, which sends the data to the Kyma-internal Loki instance. Note: This output is considered legacy and is only provided for backwards compatibility with the [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) in-cluster Loki instance. It might not be compatible with latest Loki versions. For integration with Loki, use the `custom` output with name `loki` instead.
```

Reply:
I added it as it contains the instructions on how to install it using the custom plugin. So exactly what you will need when listening to the sentence. Maybe replace , with .?

Reply:
```suggestion
    - **grafana-loki**, which sends the data to the Kyma-internal Loki instance. Note: This output is considered legacy and is only provided for backwards compatibility with the [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) in-cluster Loki instance. It might not be compatible with latest Loki versions. For integration with a custom Loki installation use the `custom` output with name `loki` instead, see also [this tutorial](https://github.com/kyma-project/examples/tree/main/loki).
```

Reply:
this sounds better. Cant think of better sentence framing.